src/components/hint/index.js

```diff
@@ -2,14 +2,15 @@
 import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
-import {StyleSheet, TouchableWithoutFeedback} from 'react-native';
+import {StyleSheet/* , TouchableWithoutFeedback */} from 'react-native';
 import {View as AnimatableView} from 'react-native-animatable';
 import {Typography, Spacings, Colors, BorderRadiuses, AnimatableManager} from '../../style';
 import {Constants} from '../../helpers';
 import {BaseComponent} from '../../commons';
 import View from '../view';
 import Text from '../text';
 import Image from '../image';
+import Modal from '../../screensComponents/modal';
 
 const sideTip = require('./assets/hintTipSide.png');
```

Comment:
The side tip doesn't render nicely with the borderRadius of the hint...

Reply:
fixed